webpack.mix.js

```diff
@@ -11,13 +11,13 @@ const mix = require('laravel-mix')
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
+mix.ts('resources/app.tsx', 'public/js')
     .react()
-    .postCss('resources/css/app.css', 'public/css', [
-        require('postcss-import'),
-        require('tailwindcss'),
-        require('autoprefixer'),
-    ])
+    // .postCss('resources/css/app.css', 'public/css', [
+    // require('postcss-import'),
+    // require('tailwindcss'),
+    // require('autoprefixer'),
+    // ])
```

Comment:
We can probably delete these lines, outright, if we don't intend to use tailwind, or any other customized CSS packages
```suggestion
```

Reply:
let's leave it for now, maybe we will define a css package later so this would be somewhat useful